### PR TITLE
Add xeno emoting

### PR DESCRIPTION
**Pull request by Vermidia**
## About the PR

All xenos can now emote! Ported the cm emotes for it too.

## Why / Balance

Parity, give xenos a tiny bit more RP potential

## Technical details

Added a server-side system just to handle the emotes, the sounds are stored on xenoComponent. Gave basexeno the emoting component, and gave xenos 5 emotes they're allowed to use.

Note I made icons for all of them, and made them colorful to match upstream's style of emotes we haven't ported yet.

Also cleaned up the felenid emote file now that punctuation for chat triggers is ignored.

## Media

https://github.com/user-attachments/assets/f36571f1-8b78-4d60-9382-5943bc9e87cc

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- fix: All xenos minus parasites can now emote properly
- fix: Xeno death noises are a bit louder now
- add: Xenos have emote sounds for roaring, growling, hissing, calling for help, or swiping their tails

### CODE REVIEW

File: Content.Shared/_RMC14/Xenonids/XenoComponent.cs [hunk 1]
```diff
@@ -75,4 +77,9 @@ public sealed partial class XenoComponent : Component
 
     [DataField, AutoNetworkedField]
     public bool SpawnAtLeaderPoint;
-}
+
+    [DataField(customTypeSerializer: typeof(PrototypeIdSerializer<EmoteSoundsPrototype>)), AutoNetworkedField]
+    public string? EmoteSounds = "Xeno";
```
Comment (deltanedas) [inline Content.Shared/_RMC14/Xenonids/XenoComponent.cs:82]: use protoid

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Dutch-VanDerLinde** [comment]
image

can't you do something similar to this for xenos instead of making a whole new system, or is the xeno marine-to-xeno chat thing breaking it?

**Comment by Vermidia** [comment]
Giving them vocal would allow them to scream and such, basically they'd get access to a ton of emotes

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

**Pull request merged**